services/wordpress/wordpress-base.js

```diff
@@ -37,6 +60,7 @@ module.exports = class BaseWordpress extends BaseJsonService {
               homepage: 0,
               tags: 0,
               screenshot_url: 0,
+              downloaded: 1,
```

Comment:
What's the purpose/impact of this field? Does it filter out plugins/themes that have 0 downloads?

Reply:
So the new API allows us to change which bits of information we need, and previously `downloaded` was returned by default. It's not anymore so we are explicitly asking for that bit of information. 

`1` is us requesting it
`0` is us telling the API to skip that

That block currently skips a lot of info we don't need, but can also be expanded to request additional information. Some of the new badges I will be submitting will also add to this block to request the information that they need. A similar change was made to the platform tester file, as it changes the expected url. 

It won't skip anything that has 0 downloads, but will just ensure we get that field returned in the response.

Reply:
Ah okay cool, thanks for the info (it actually makes more sense now that i think about it being a member of `fields` :laughing:)